src/box/error.h

```diff
@@ -203,6 +203,11 @@ class ClientError: public Exception
 		throw this;
 	}
 
+	virtual struct error *dup() const
+	{
+		return new ClientError;
```

Comment:
Confusing function name. The function doesn't duplicate the error - it just allocates one. So it should either be named accordingly (alloc?) or actually duplicate the error content.

Reply:
Now there is a `copy()` method that calls an appropriate copy constructor.
Shouldn't it be resolved? If you don't agree please explain the reasons.